src/Culling/babylon.boundingBox.ts

```diff
@@ -69,7 +72,7 @@
         // Methods
 
         /**
-         * Recreates the entire bounding box from scratch
+         * Recreates the entire bounding box from scratch, producing same values as if the constructor was called.

```

Comment:
looks like a breaking change to me

Reply:
is it really unwanted as the comment said that it recreates the entine BBox from scratch ?
Also, from within the engine, it seems that the usual way to compute the world woordinates boundingBox after construction or reconstruction is to call update, that's why computing them from reconstruct is kind of extra-useless computation.

Reply:
It is a public API so we do not know if people use it differently hence we cannot change the way it works

Reply:
ok, I'll add another function that do what I want so that I don't change the API.

Reply:
Also it is strange that's the behavior `reConstruct` is not the same between BoundingBox and BoundingSphere